src/Sparrow/LowMemory/MemoryInformation.cs

```diff
@@ -182,14 +209,27 @@ public static (Size MemAvailable, Size TotalMemory, Size Commited, Size CommitLi
                     var totalMemInKb = bufferedReader.ExtractNumericValueFromKeyValuePairsFormattedFile(MemTotal);
                     var swapTotalInKb = bufferedReader.ExtractNumericValueFromKeyValuePairsFormattedFile(SwapTotal);
                     var commitedInKb = bufferedReader.ExtractNumericValueFromKeyValuePairsFormattedFile(Committed_AS);
+
+                    var calculatedInBytes = new Size(memAvailableInKb, SizeUnit.Kilobytes);
+                    var sharedCleanMemory = new Size(0, SizeUnit.Bytes);
+                    if (smapsReader != null)
+                    {
+                        var result = smapsReader.CalculateMemUsageFromSmaps<SmapsReaderNoAllocResults>();
+                        calculatedInBytes.Add(result.SharedClean, SizeUnit.Bytes);
+                        calculatedInBytes.Add(result.PrivateClean, SizeUnit.Bytes);
+                        sharedCleanMemory.Set(result.SharedClean, SizeUnit.Bytes);
+                    }
+
                     return (
                         MemAvailable: new Size(Math.Max(memAvailableInKb, memFreeInKb), SizeUnit.Kilobytes),
                         TotalMemory: new Size(totalMemInKb, SizeUnit.Kilobytes),
                         Commited: new Size(commitedInKb, SizeUnit.Kilobytes),
 
                         // on Linux, we use the swap + ram as the commit limit, because the actual limit
                         // is dependent on many different factors
-                        CommitLimit: new Size(totalMemInKb + swapTotalInKb, SizeUnit.Kilobytes)
+                        CommitLimit: new Size(totalMemInKb + swapTotalInKb, SizeUnit.Kilobytes),
+                        CalculatedAvailableMemory: calculatedInBytes,
+                        SharedCleanMemory: sharedCleanMemory
```

Comment:
I don't like the name, better to call it `TotalClean`, as that is what it seems to be doing.